taskcat/cli.py

```diff
@@ -1,156 +1,77 @@
-import importlib
+import argparse
 import signal
 import sys
-from pathlib import Path
-from typing import List
 
 import pyfiglet
 import requests
 from pkg_resources import get_distribution
 
 import taskcat
+import taskcat.cli_modules
 from taskcat.cli_core import CliCore
-from taskcat.common_utils import exit0, exit1
+from taskcat.common_utils import exit_with_code
```

Comment:
You missed a few...

```
 …/qstools/taskcat-v9     lint   7  egrep -i 'exit[0-9]' -R *
build/lib/taskcat/cli.py:from taskcat.common_utils import exit0, exit1
build/lib/taskcat/cli.py:        exit1()
build/lib/taskcat/cli.py:        exit1()
build/lib/taskcat/cli.py:    exit0()
build/lib/taskcat/cli.py:        exit1("--debug and --quiet cannot be specified simultaneously")
build/lib/taskcat/cli.py:    exit1()
build/lib/taskcat/common_utils.py:def exit1(msg=""):
build/lib/taskcat/common_utils.py:def exit0(msg=""):
build/lib/taskcat/legacy_cli.py:from taskcat.common_utils import exit0, exit1
build/lib/taskcat/legacy_cli.py:                exit1("Lambda build not enabled for project. Add package-lambda: true to taskcat.yaml global section")
build/lib/taskcat/legacy_cli.py:                    exit0("Lambda source zipped successfully")
build/lib/taskcat/legacy_cli.py:                exit1(str(e))
build/lib/taskcat/legacy_cli.py:                exit0("Linting completed")
build/lib/taskcat/legacy_cli.py:                exit1("One or more tests failed. See the report for details.")
build/lib/taskcat/legacy_cli.py:        exit1()
build/lib/taskcat/legacy_cli.py:        exit1()
build/lib/taskcat/legacy_cli.py:        exit0()
build/lib/taskcat/legacy_cli.py:        exit0()
build/lib/taskcat/legacy_cli.py:    exit1()
build/lib/taskcat/stacker.py:from taskcat.common_utils import exit0, exit1, param_list_to_dict
build/lib/taskcat/stacker.py:            exit1("Cannot locate file %s" % config_yml)
build/lib/taskcat/stacker.py:            exit0("Upload completed successfully")
build/lib/taskcat/stacker.py:            exit1(
build/lib/taskcat/stacker.py:                    exit1()
build/lib/taskcat/utils.py:from taskcat.common_utils import exit1
build/lib/taskcat/utils.py:            exit1()
taskcat/legacy_cli.py:from taskcat.common_utils import exit0, exit1
taskcat/legacy_cli.py:                exit1("Lambda build not enabled for project. Add package-lambda: true to taskcat.yaml global section")
taskcat/legacy_cli.py:                    exit0("Lambda source zipped successfully")
taskcat/legacy_cli.py:                exit1(str(e))
taskcat/legacy_cli.py:                exit0("Linting completed")
taskcat/legacy_cli.py:                exit1("One or more tests failed. See the report for details.")
taskcat/legacy_cli.py:        exit1()
taskcat/legacy_cli.py:        exit1()
taskcat/legacy_cli.py:        exit0()
taskcat/legacy_cli.py:        exit0()
taskcat/legacy_cli.py:    exit1()
taskcat/stacker.py:            exit1("Cannot locate file %s" % config_yml)
taskcat/stacker.py:            exit0("Upload completed successfully")
taskcat/stacker.py:            exit1(
taskcat/stacker.py:                    exit1()
```

Reply:
don't think that grep was run against my pr :D when I search I only find 4 references in `stacker.py` which I don't plan on applying fixes like this to because it is all destined to be refactored anyways.